assets/js/blocks/cart-checkout/checkout/form/shipping-options-step.tsx

```diff
@@ -50,7 +54,7 @@ const renderShippingRatesControlOption = ( option ) => {
 	};
 };
 
-const ShippingOptionsStep = () => {
+const ShippingOptionsStep = (): JSX.Element | null => {
```

Comment:
JSX.Element vs ReactNode - which is better? `ReactNode` covers null types.

Reply:
I initially started with ReactNode, but then ran into issues like this:

![image](https://user-images.githubusercontent.com/5656702/116405155-3c12fb80-a827-11eb-8eb9-946dc6ce1284.png)

Also, when the return type is not explicitly specified, TypeScript infers that it is is a `JSX.Element`. I would prefer to stick with that on the basis that it doesn't break, and TS thinks it's that anyway.